src/App.js

```diff
@@ -1,15 +1,12 @@
 import React, {Component} from "react";
```

Comment:
In react, from version 17.0.0 and above, `import react from 'React'` is no longer needed for components to render :)

(Of course, named imports like `useState` are needed as in every other module...)